content/hints.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Advice, Hints, and General Information from the NWERC Jury"
+weight: 6
+draft: false
+menu:
+  main:
+    name: Hints
```

Comment:
Rather then Hints, shouldn't it be Judging Notes? They are more notes on how the problems then hints how to solve them.

Reply:
They're hints to understand the problem _format_, not how to solve them, but I can see where you're coming from. I've renamed the page to Jury Advice :slightly_smiling_face: